cve_bin_tool/cli.py

```diff
@@ -197,28 +194,24 @@ def scan_file(self, filename):
                         version = result["version"]
                     elif result["version"] == "UNKNOWN":
                         self.logger.warning(
-                            "%r was detected with version UNKNOWN in file %r"
-                            % (result["modulename"], filename)
+                            f"{modulename} was detected with version UNKNOWN in file {filename}"
```

Comment:
I was worried this should have been result["modulename"] but there is indeed a modulename variable.  I wonder why we didn't use it in the message before?